src/ImageResizerServiceProvider.php

```diff
@@ -16,10 +16,24 @@ class ImageResizerServiceProvider extends ServiceProvider
      */
     public function boot()
     {
-        Route::get('storage/resizes/{size}/{file}{webp?}', ImageController::class)
-            ->where(['file' => '.*\.((?!webp)[^\.])+', 'webp' => '\.webp'])
+        Route::get('storage/{store}/resizes/{size}/{file}{webp?}', ImageController::class)
+            ->where([
+                'store' => '[0-9]*',
+                'file'  => '.*\.((?!webp)[^\.])+',
+                'webp'  => '\.webp',
+            ])
             ->name('resized-image');
 
+        // Backwards compatibility step.
+        Route::get(
+            'storage/resizes/{size}/{file}{webp?}',
+            fn (string $size, string $file, string $webp) => redirect(route('resized-image', ['store' => config('rapidez.store'), 'size' => $size, 'file' => $file, 'webp' => $webp]), 301)
+        )
+        ->where([
+            'file' => '.*\.((?!webp)[^\.])+',
+            'webp' => '\.webp',
+        ]);
```

Comment:
I think this is a cleaner syntax:
```
Route::get('storage/resizes/{size}/{file}{webp?}', function (string $size, string $file, string $webp) {
    return redirect(route('resized-image', ['store' => config('rapidez.store'), 'size' => $size, 'file' => $file, 'webp' => $webp]), 301);
})->where([
    'file' => '.*\.((?!webp)[^\.])+',
    'webp' => '\.webp',
]);
```

Or with `compact()`:
```
Route::get('storage/resizes/{size}/{file}{webp?}', function (string $size, string $file, string $webp) {
    return redirect(route('resized-image', ['store' => config('rapidez.store')] + compact('size', 'file', 'webp')), 301);
})->where([
    'file' => '.*\.((?!webp)[^\.])+',
    'webp' => '\.webp',
]);
```

Or maybe even better to extract the route patterns as they're used twice:
```
$patterns = [
    'store' => '[0-9]*',
    'file' => '.*\.((?!webp)[^\.])+',
    'webp' => '\.webp',
];

Route::get('storage/resizes/{size}/{file}{webp?}', function (string $size, string $file, string $webp) {
    return redirect(route('resized-image', ['store' => config('rapidez.store')] + compact('size', 'file', 'webp')), 301);
})->where($patterns);
```